### PR TITLE
reissue cluster certificates every 90 days

### PR DESCRIPTION
**Pull request by jim-minter**

### CODE REVIEW

File: pkg/install/install.go [hunk 1]
```diff
@@ -155,12 +155,10 @@ func (i *Installer) AdminUpgrade(ctx context.Context) error {
 		action(i.fixLBProbes),
 		action(i.fixPullSecret),
 		action(i.ensureGenevaLogging),
+		action(i.upgradeCertificates),
```
Comment (m1kola) [inline pkg/install/install.go:158]: Do we `upgradeCertificates` step here permanently or Is it going to be on-off admin update? I'm a bit worried about `LifetimeActions` getting out of sync.

Comment (jim-minter) [reply pkg/install/install.go:158]: I'm hoping it's going to be one-off.  What's your concern about LifetimeActions?

File: pkg/deploy/predeploy.go [hunk 2]
```diff
@@ -77,7 +77,7 @@ func (d *deployer) PreDeploy(ctx context.Context) error {
 		}
 	}
 
-	return d.ensureMonitoringCertificates(ctx)
+	return nil
```
Comment (mjudeikis) [inline pkg/deploy/predeploy.go:80]: What changed here? Why we dont need this anymore?

File: pkg/install/install.go (hunk 1 above)
Comment (m1kola) [reply pkg/install/install.go:158]: `LifetimeActions` in `CreateSignedCertificate` and in `UpgradeCertificatePolicy` should have the same values. If it's a one-off thing - let's maybe create a follow-up issue to remove the function.

If we want to keep this function - maybe let's make the connection more obvious in the code. E.g. by defining a constant and using it in both places or something like this. To make it harder to change value in one place but forget the other?

We probably not going to change it often, but we might want to reduce the time at some point.

File: pkg/deploy/predeploy.go (hunk 2 above)
Comment (jim-minter) [reply pkg/deploy/predeploy.go:80]: Goal is to get these certs populated and auto-renewed by key vault.

Comment (jim-minter) [reply pkg/deploy/predeploy.go]: Moved this part of the PR to https://github.com/Azure/ARO-RP/pull/754

**Pull request merged**